Duckling/Temperature/EN/Corpus.hs

```diff
@@ -52,6 +53,13 @@ allExamples = concat
              , "2 degrees below zero"
              , "2 below zero"
              ]
+  , examples (simple Kelvin 44)
```

Comment:
these examples look good, classifiers need to be regenerated

Reply:
I've run 
`stack build :duckling-regen-exe && stack exec duckling-regen-exe && stack test`

And I can parse out kelvins.
```
curl -XPOST http://0.0.0.0:8000/parse --data 'locale=en_GB&text=44 kelvin'        
[{"body":"44 kelvin","start":0,"value":{"value":44,"type":"value","unit":"kelvin"},"end":9,"dim":"temperature","latent":false}]
curl -XPOST http://0.0.0.0:8000/parse --data 'locale=en_GB&text=11 kelvins'
[{"body":"11 kelvins","start":0,"value":{"value":11,"type":"value","unit":"kelvin"},"end":10,"dim":"temperature","latent":false}]
```

The only classifier that updated was `IT_XX.hs` 🤷🏽 I was expecting `EN_XX.hs` to update.